backends/rapidpro/backend_test.go

```diff
@@ -715,6 +715,13 @@ func (ts *BackendTestSuite) TestCheckForDuplicate() {
 	msg1 := createAndWriteMsg(knChannel, urn, "ping", "")
 	ts.False(msg1.alreadyWritten)
 
+	keys, err := redis.Strings(r.Do("KEYS", "seen-msgs:*"))
+	ts.NoError(err)
+	ts.Len(keys, 1)
```

Comment:
looking at this key made me realize we need https://github.com/nyaruka/redisx/pull/2